documentation/airport-guides/egll.md

```diff
@@ -0,0 +1,67 @@
+### About London Heathrow
+
+Heathrow Airport (Also known as London Heathrow) (ICAO: LKPR, IATA: LHR) is the main and biggest airport in the United Kingdom, acting as the primary hub for British Airways (ICAO: BAW, callsign: SPEEDBIRD), and the primary operating airport for Virgin Atlantic (ICAO: VIR, callsign: VIRGIN). 
+
+Heathrow Airport serves the majority of London air travel, as well as the majority of southern Britain. It is located 14 miles to the West of London's City Center. It is one of the six london airports servicing London (Along with City, Stansted, Luton, Southend and Gatwick). 
+
+Heathrow airport started in 1929 as a small airfield in the hamlet of Heathrow. The development of Heathrow picked up pace in 1944 as a stopover location for long-distance military aircraft, however, by the time it neared completion WW2 had already finished, and the British government continued to develop the airport for civil use. It was opened on 25 March 1946 named 'London Airport', and renamed to 'Heathrow Airport' in 1966. 
```

Comment:
Change to:
`Heathrow Airport was first built in 1929 as a small airfield in the hamlet of Heathrow. The development of Heathrow picked up pace in 1944 as a stopover location for long-distance military aircraft, however, by the time it neared completion, WW2 had already finished, so the British government continued to develop the airport for civil use. It was  finally opened in 1946 as 'London Airport', and renamed to 'Heathrow Airport' in 1966. `